src/Modules/Admin/Commands.Admin/help/Set-PowerBIEncryptionKey.md

```diff
@@ -0,0 +1,75 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Set-PowerBIEncryptionKey
+
+## SYNOPSIS
+Rotate the encryption key
```

Comment:
Mention product. Mention Set here instead of rotate and in the description use Rotates